IFC4x3/Properties/e/EarthquakeSensorRange_3IBX9s4kz9agU5ycgPD6BR/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicates the measuring range of the earthquake sensor or accelerometer.
```

Comment:
How is this different from all the other ranges specified?

Could DetectionRange not serve this purpose in the context of the pset?

Reply:
No, the data type is different.